src/lib/alloc.c

```diff
@@ -875,6 +875,8 @@ static void *alloc_heap_buffer(struct mm_heap *heap, uint32_t flags,
 	return ptr;
 }
 
+#define USE_UNCACHE_BUFFER_ONLY
```

Comment:
This needs to be a Kconfig option i.e. FORCE allocator to use coherent memory only - and explain it's a debug option.

Reply:
> This needs to be a Kconfig option i.e. FORCE allocator to use coherent memory only - and explain it's a debug option.

Okay, will do. Do we want to make this "use coherent memory only" selected by default at the moment? As without it, we are hitting kinds of Xrun issues which may annoy the CI a lot.